src/intl/en.json

```diff
@@ -155,5 +155,49 @@
+  "page-find-wallet-use-wallet-desc": "Now that you have a wallet, check out some Ethereum applications (dapps). There are dapps for finance, social media, gaming and lots of other categories.",
+  "page-find-wallet-checkout-dapps": "Check out dapps",
+  "page-find-wallet-buy-card": "Buy crypto with a card",
+  "page-find-wallet-buy-card-desc": "Buy ETH directly from your wallet with a bank card. Geographical restrictions may apply.",
+  "page-find-wallet-no-fees": "No transaction fees",
+  "page-find-wallet-no-fees-desc": "Your wallet will pay some of your transaction fees for you. This may only be for certain types of transactions and restrictions may apply.",
+  "page-find-wallet-explore-dapps": "Explore dapps",
+  "page-find-wallet-explore-dapps-desc": "These wallets are designed to help you connect to Ethereum dapps.",
+  "page-find-wallet-fi-tools": "Access to financial tools",
+  "page-find-wallet-fi-tools-desc": "Borrow, lend and earn interest directly from your wallet.",
+  "page-find-wallet-withdraw": "Withdraw to bank",
+  "page-find-wallet-withdraw-desc": "You can cash out your ETH straight to your bank account without going through an exchange.",
+  "page-find-wallet-limits": "Limits protection",
+  "page-find-wallet-limits-desc": "Safeguard your assets by setting limits that prevent your account being drained.",
+  "page-find-wallet-volume": "High-volume purchases",
+  "page-find-wallet-voluem-desc": "If you want to hold a lot of ETH, choose a wallet that lets you buy more than $2000 ETH at a time.",
+  "page-find-wallet-swaps": "Decentralized token swaps",
+  "page-find-wallet-swaps-desc": "Trade between ETH and other tokens directly from your wallet.",
+  "page-find-wallet-multisig": "Multi-signature accounts",
+  "page-find-wallet-multisig-desc": "For extra security, multi-signature wallets require more than one account to authorise transactions.",
+  "page-find-wallet-feature-h2": "Choose the wallet features you care about",
+  "page-find-wallet-search-btn": "Search chosen features",
+  "page-find-wallet-Ethereum-wallets": "Ethereum Wallets",
+  "page-find-wallet-we-found": "We found",
+  "page-find-wallet-following-features": "with the following features:",
+  "page-find-wallet-showing": "Showing ",
+  "page-find-wallet-overwhelmed": "Ethereum wallets below. Overwhelmed? Try filtering by features.",
+  "page-find-wallet-clear": "Clear filters",
+  "page-find-wallet-not-all-features": "No wallet has all of these features",
+  "page-find-wallet-yet": "yet",
+  "page-find-wallet-try-removing": "Try removing a feature or two",
+  "page-find-wallet-not-endorsements": "Wallets listed on this page are not official endorsements, and are provided for informational purposes only. Their descriptions have been provided by the wallet companies themselves. We add products to this page based on criteria in our",
+  "page-find-wallet-listing-policy": "listing policy",
+  "page-find-wallet-add-wallet": ". If you'd like us to add a wallet,",
```

Comment:
@ph5500 again just a small note - don't start/end phrases with spaces. Instead, add the period & the space within the code.